pages/Ancient-Pedestal.md

```diff
@@ -0,0 +1,11 @@
+The Ancient Pedestal is a block in the [Magical Items](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Items) category. It is essential in building a working [Ancient Altar](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ancient-Altar).
+
+## Obtaining
+The Ancient Pedestal can be crafted in a [Magic Workbench](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magic-Workbench).
+
+## Usage
+8 Ancient Pedestals must be placed around an [Ancient Altar](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ancient-Altar) in order for it to work.
+
+Right-clicking a pedestal with an item in hand will make it rotate and levitate above the pedestal, which may be useful for decorative purposes.
```

Comment:
You should mention that the item will despawn after 5 minutes, like a dropped item.